package.json

```diff
@@ -15,6 +16,6 @@
     "mocha-junit-reporter": "1.6.1"
   },
   "scripts": {
-    "test": "grunt verify --force"
+    "test": "./node_modules/mocha/bin/mocha test/generic.spec.js"
```

Comment:
https://github.com/Real-Skill/support/wiki/I-want-to-create-my-own-task#keep-it-multiplatform